cloudant-client/src/main/java/com/cloudant/client/api/scheduler/SchedulerDocsResponse.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright © 2018 IBM Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+package com.cloudant.client.api.scheduler;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+public class SchedulerDocsResponse {
```

Comment:
I wonder if it is worth this (and its friend `SchedulerJobsResponse`) extending a `SchedulerResponse` class that contains the `offset` and `total_rows` fields and methods.

It would be nice to take it even further and do `SchedulerResponse<T>` and typing with `Doc` or `Job` as needed, but sadly the `getDocs` `getJobs` method naming would be a bit unfriendly then.

Reply:
Well there are other things which `offset` and `total_rows`, like views, as well.